public-key.asc

```diff
@@ -0,0 +1,42 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
```

Comment:
For what usecase do we need a public key? How is it generated? Who owns the private key?

Reply:
my local setup, I need to use PGP Key... I accidentally added it..